src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildANetCoreApp.cs

```diff
@@ -800,6 +801,28 @@ public void ItHasNoPackageReferences()
                 .BeEmpty();
         }
 
+        [WindowsOnlyFact]
+        public void ItResolvesPackageAssetsMultiTargetingNetStandard()
+        {
+            var testProject = new TestProject()
+            {
+                Name = "MultiTargetedPackageReference",
+                TargetFrameworks = ToolsetInfo.CurrentTargetFramework +";netstandard2.1",
+                RuntimeIdentifier = "win-x64",
+                IsExe = true
+            };
+            testProject.PackageReferences.Add(new TestPackageReference("Nuget.Common","6.5.7"));
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject, testProject.Name);
+
+            var buildCommand = new BuildCommand(testAsset);
+
+            buildCommand
+                .Execute()
+                .Should()
+                .Pass();
```

Comment:
It's probably worth checking that the correct NU1603 warinng is generated

```suggestion
                .Pass()
                .And
                .HaveStdOutContaining("NU1603");
```